packages/angular/src/generators/remote/remote.ts

```diff
@@ -36,5 +36,70 @@ export default async function remote(tree: Tree, options: Schema) {
     port: options.port ?? findNextAvailablePort(tree),
   });
 
+  removeDeadCode(tree, options);
+
   return installTask;
 }
+
+function removeDeadCode(tree: Tree, options: Schema) {
+  const project = readProjectConfiguration(tree, options.name);
+
+  ['css', 'less', 'scss', 'sass'].forEach((style) => {
+    const pathToComponentStyle = joinPathFragments(
+      project.sourceRoot,
+      `app/app.component.${style}`
+    );
+    if (tree.exists(pathToComponentStyle)) {
+      tree.delete(pathToComponentStyle);
+    }
+  });
+
+  tree.delete(
+    joinPathFragments(project.sourceRoot, 'app/nx-welcome.component.ts')
+  );
+  tree.delete(
+    joinPathFragments(project.sourceRoot, 'app/app.component.spec.ts')
+  );
+  tree.delete(joinPathFragments(project.sourceRoot, 'app/app.component.html'));
+
+  const pathToComponent = joinPathFragments(
+    project.sourceRoot,
+    'app/app.component.ts'
+  );
+  const component =
+    tree.read(pathToComponent, 'utf-8').split('templateUrl')[0] +
+    `template: '<router-outlet></router-outlet>'
+})
+export class AppComponent {}`;
```

Comment:
imho, it would be better if we have the full template for the component here rather than splitting by a property based on what was previously generated. It's a small component, so the template would be small. It should be clearer, and we'd also reduce the potential maintenance associated with relying on a previous generation that could change and that we don't need to rely on. Regardless of a previous generation, we know the content of what we want in this component.

Reply:
Very good point. I will add this as an additional cleanup PR